tests/core/test_expectation_suite.py

```diff
@@ -267,6 +269,7 @@ def test_expectation_suite_equivalence(
     assert not single_expectation_suite.isEquivalentTo(baseline_suite)
 
 
+@pytest.mark.unit
 def test_expectation_suite_dictionary_equivalence(baseline_suite):
     assert (
         baseline_suite.isEquivalentTo(
```

Comment:
there is a lot of duplication in this test body, can we reuse the expectation definition? It'll also make the change more obvious when we make it (in the false case)

Reply:
Also, it seems like we are foregoing using the fixtures we have